tapfreighter/chain_porter.go

```diff
@@ -527,7 +549,8 @@ func (p *ChainPorter) fetchInputProof(ctx context.Context,
 		ScriptKey: *scriptKey,
 		OutPoint:  &input.OutPoint,
 	}
-	inputProofBytes, err := p.cfg.AssetProofs.FetchProof(
+	log.Infof("fetching proof for input: ", spew.Sdump(inputProofLocator))
```

Comment:
nit: degrade to `log.Tracef`?

Reply:
Actually a left over debug log, will just remove.